### PR TITLE
Improve logging

### PR DESCRIPTION
**Pull request by denis-tingajkin**
Signed-off-by: denis-tingajkin <denis.tingajkin@xored.com>

## Motivation and Context
https://github.com/networkservicemesh/networkservicemesh/issues/1053

## How Has This Been Tested?

- [X] Covered by existing integration testing
- [ ] Added integration testing to cover
- [X] Tested locally
- [ ] Have not tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: test/integration/my_tag_test.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+package nsmd_integration_tests
+
+import (
+	"testing"
+
+	"github.com/networkservicemesh/networkservicemesh/test/kubetest"
+	. "github.com/onsi/gomega"
+)
+
+func TestWhyMyTestFail(t *testing.T) {
```
Comment (denis-tingajkin) [inline test/integration/my_tag_test.go]: TODO: Remove demo test

**Comment by denis-tingajkin** [comment]
now logs from pods saves in Artifacts on circleci
https://circleci.com/gh/networkservicemesh/networkservicemesh/76791#artifacts/containers/0

File: test/cloudtest/pkg/utils/utils.go [hunk 2]
```diff
@@ -9,3 +15,14 @@ func Contains(a []string, x string) bool {
 	}
 	return false
 }
+
+// NewRandomStr - generates random string of desired length, size should be multiple of two for best result.
```
Comment (nickolaev) [inline test/cloudtest/pkg/utils/utils.go]: Could this be useful to the borader project? Shall we move it to the root utils?

Comment (haiodo) [reply test/cloudtest/pkg/utils/utils.go]: I suppose finally yes, but for separate project it should be at least documented and finally

**Pull request merged**